### PR TITLE
Add generation of crc-bundle-info.json file

### PR DESCRIPTION
**Pull request by cfergeau**
This PR is just a single commit, but it's on top of the changes already proposed in the PR #7 
This should get clearer once #7 is merged.
The related issue ( https://github.com/code-ready/crc/issues/34 ) has some open comment in https://github.com/code-ready/crc/issues/34#issuecomment-485845805 :

What I'm generating so far:
```
{
  "type": "snc",
  "buildInfo": {
    "buildTime": "2019-04-23T14:55:32+00:00",
    "openshiftInstallerVersion": "./openshift-install v0.16.1\nbuilt from commit e3fceacc975953f56cb09931e6be015a36eb6075",                                                                                       
    "sncVersion": "git9662"
  },
  "clusterInfo": {
    "clusterName": "crc",
    "baseDomain": "tt.testing",
    "sshPrivateKeyFile": "master_privatekey",
    "kubeConfig": "kubeconfig"
  },
  "storage": {
    "diskImages": [
      {
        "name": "crc.qcow2",
        "format": "qcow2"
      }
    ]
  }
}
```
The "diskImages" bit should probably be made simpler for now, it's not clear at all to me right now how we would describe multi node setups in this file. Not sure it would be convenient at all.

### CODE REVIEW

File: crc-bundle-info.json.sample [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    "openshiftInstallerVersion": "./openshift-install v0.16.1\nbuilt from commit e3fceacc975953f56cb09931e6be015a36eb6075",
+    # Output of 'git describe' or 'git rev-parse' of the 'snc' script
+    # repository used when generating the bundle
+    "sncVersion": "git9662"
+  },
+  "clusterInfo": {
+    # Name of the openshift cluster stored in the bundle
+    "clusterName": "crc",
+    # Base domain name used for the openshift cluster
+    "baseDomain": "tt.testing",
```
Comment (praveenkumar) [inline crc-bundle-info.json.sample]: We also need the hostname which can be `${VM_PREFIX}-master-0` in our case.

Comment (cfergeau) [reply crc-bundle-info.json.sample]: This has been added in the latest iterations.

File: createdisk.sh [hunk 2]
```diff
@@ -29,10 +29,22 @@ function create_disk_image {
     rm -fr $tarballDirectory/${VM_PREFIX}-master-0 $tarballDirectory/${VM_PREFIX}-base
 }
 
+function update_json_description {
+    cat $1/crc-bundle-info.json \
+        | ${JQ} ".clusterInfo.masterHostname = \"${VM_PREFIX}-master-0\"" \
```
Comment (praveenkumar) [inline createdisk.sh]: @cfergeau you added it here but your sample file still doesn't have a field for `masterHostname` under `clusterInfo`

File: createdisk.sh [hunk 3]
```diff
@@ -29,10 +29,22 @@ function create_disk_image {
...
+        | ${JQ} '.clusterInfo.sshPrivateKeyFile = "master_privatekey"' \
```
Comment (praveenkumar) [inline createdisk.sh]: @cfergeau shouldn't it would be now `id_rsa_crc` ?

File: createdisk.sh (hunk 2 above)
Comment (cfergeau) [reply createdisk.sh]: Ah indeed, thanks for catching this!

File: createdisk.sh (hunk 3 above)
Comment (cfergeau) [reply createdisk.sh]: I kept the initial name, but I definitely prefer id_rsa_crc :)

**Pull request merged**